src/traffic_ctl/config.cc

```diff
@@ -51,6 +55,8 @@ struct CtrlMgmtRecordDescriptionList : CtrlMgmtList<RecordDescriptionPolicy> {
   }
 };
 
+#include "../shared/txn_vars.cc"
```

Comment:
This is slightly wonky, do we really need / want to include .cc files ? Seeing this, maybe src/shared should compile to a .a (not .so), and we add a include/shared for the corresponding prototypes?

Reply:
Hahaha, the bike shedder arrives!

I'm not sure. Is it worth the set up for a single file?

In any case, I don't like the ".a" approach. If you want a change, I would create a header file, make the `Overridable_Map` not static, and grab the file via "Makefile.inc" reaching over to "../shared/txn-vars.cc".

What would the directory structure be?
-shared
  -src
    -txn_vars.cc
  -include
    -txn_vars.h

or
-shared
  -txn_vars.cc
  -include
    -txn_vars.h